requirements/run.txt

```diff
@@ -1,71 +1,72 @@
 #
-# This file is autogenerated by pip-compile with Python 3.9
+# This file is autogenerated by pip-compile with Python 3.11
 # by the following command:
 #
-#    pip-compile --output-file=requirements/run.txt --resolver=backtracking requirements/run.in
+#    pip-compile --output-file=requirements/run.txt requirements/run.in
 #
-anyio==3.6.2
+anyio==4.3.0
     # via
-    #   httpcore
+    #   httpx
     #   starlette
     #   watchfiles
-asgiref==3.6.0
+asgiref==3.7.2
     # via -r requirements/run.in
-asttokens==2.0.8
+asttokens==2.4.1
     # via stack-data
 attrs==21.4.0
     # via jsonschema
-backcall==0.1.0
-    # via ipython
-certifi==2021.10.8
+certifi==2024.2.2
```

Comment:
It's OK and in fact great to also upgrade the transitive requests/httpx dependencies `certifi`, `urllib3`, `idna` and `charset-normalizer`. So, please also send a PR upgrading the NApps like `mef_eline` https://github.com/kytos-ng/mef_eline/blob/master/requirements/run.txt#L15, and the other ones that have `requests` dependencies. Since this is just transitive simple dependencies you can just bump directly on `requirements/run.txt` without regenerating on NApps that it should still work (normally that's not the way we do it, but since we know that these are terminal/leaf node dependencies it should be OK)

Reply:
Creating PRs